include/swift/Frontend/FrontendInputsAndOutputs.h

```diff
@@ -143,11 +133,10 @@ class FrontendInputsAndOutputs {
 
   const InputFile &getRequiredUniquePrimaryInput() const;
 
-  /// \return the name of the unique primary input, or an empty StrinRef if
-  /// there isn't one.
-  StringRef getNameOfUniquePrimaryInputFile() const;
-
-  /// Combines all primaries for stats reporter
+  /// Should combine all primaries for stats reporter
+  /// Today, just answers "batch" if > 1, so it could
+  /// return a StringRef, but someday will have to fabricate
+  /// a temporary string.
```

Comment:
It doesn't *currently* return a StringRef, so the comment doesn't make sense. (It is still something we have to fix, but picking the first filename is probably good enough.)

Reply:
There exists an interpretation of the words that is what I meant to say, but I shortened the comment and removed the whole issue of return types. I added "FIXME:".